specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/azureFirewall.json

```diff
@@ -838,12 +851,23 @@
             "type": "string"
           }
         },
-        "destinationFqdns": {
+        "sourceIpGroups": {
```

Comment:
Please add new properties at the end (looks like they should go after `destinationFqdns`)